rocketpy/mathutils/function.py

```diff
@@ -2817,6 +2817,76 @@ def compose(self, func, extrapolate=False):
                 extrapolation=self.__extrapolation__,
             )
 
+    def savetxt(
+        self,
```

Comment:
Actually, just saw what you commented in the description. I still think it could be more descriptive to use `to_` names

Reply:
This is an interesting suggestion.

Between numpy and pandas, I have to say that numpy might be a better inspiration in this case, since pandas is not one of the rocketpy's dependency.

Beyond that, I believe we need to be careful with the `to_` methods, let's see:
- `.savetxt`: takes a Function instance and save its source to a text file, where each row represents one point from the source.
- `.savecsv`: this call the savetxt method above but modify the file extension to .csv
- `.to_json` (or other similar name like ``) takes the Function instance and create a json file with different attributes from the object (e.g. interpolation method, inputs and outputs names, repr method, etc.)

If we change the `savetxt` to `to_txt` we would be creating an opportunity for confusion since the `to_txt` and `to_json` would export totally different contents.
There's a reason for having the `save` methods to save the source and the `to_` methods to convert the *entire object* to something else.

Let me know in case your disagree with this.

The #522 might be connected to this comment too.